docs/pages/machine-id/guides/databases.mdx

```diff
@@ -0,0 +1,284 @@
+will be able to get the same short-lived certificates tied to a machine
+identity that can be rotated, audited, and controlled with all the familiar
+Teleport access controls.
+
+<Figure align="left" bordered caption="Machine ID and Database Access Deployment">
+  ![Machine ID and Database Access Deployment](../../../img/machine-id/machine-id-database-app.png)
+</Figure>
+
+## Prerequisites
+
+You will need Teleport  >= 9.3.0 
+
+If you have not already put your database behind Teleport Database Access,
+follow the [Database Access Getting Started Guide](../../database-access/getting-started.mdx).
+Database Access supports databases like PostgreSQL, MongoDB, Redis, and much
+more. See our [Database Access Guides](../../database-access/guides.mdx) for
+a complete list.
+
+If you have not already setup Machine ID, follow the
+[Machine ID Getting Started Guide](../getting-started.mdx)
+to familiarize yourself with Machine ID.
+
+Lastly, ensure both the `tbot` and `tsh` executables are available on your
+desired client system.
+
+## Step 1/3. Create a Machine ID bot and assign permissions
+
+In the example below, you will create a bot user named `app` and assign
+this bot user the `machine-id-db` role.
+
+First, create a role that Machine ID can use to access your database:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: machine-id-db
+spec:
+  allow:
+    db_labels:
+      '*': '*'
+    db_names: [example]
+    db_users: [alice]
+    rules:
+      - resources: [db_server, db]
+        verbs: [read, list]
+```
+
+This role allows Machine ID bots to do two things:
+ 1. Access the database `example` on any database server (due to the `'*': '*'`
+    label selector) as the user `alice`. You may restrict access to the bot
+    using a more specific label selector; see the [Database Access RBAC 
+    guide](../../database-access/rbac.mdx) for more info.
+ 2. Discover information about databases in Teleport.
+
+Write this to `role.yaml` and run the following to create the role in Teleport:
+
+```code
+$ tctl create -f role.yaml
+```
+
+With the role created, create a new bot and allow it to assume the new role.
+
+<Details scope={["cloud"]} scopeOnly={true}>
+On your client machine, log in to Teleport using `tsh` before using `tctl` to
+create the bot:
+
+```code
+$ tctl bots add app --roles=machine-id-db
+```
+</Details>
+<Details scope={["oss","enterprise"]} scopeOnly={true}>
+Connect to the Teleport Auth Server and use `tctl` to create the bot:
+
+```code
+$ tctl bots add app --roles=machine-id-db
+```
+</Details>
+
+## Step 2/3. Configure and start Machine ID
+
+Next, we'll run Machine ID alongside our database client app to begin fetching
+credentials.
+
+Start by creating a configuration file for Machine ID at `/etc/tbot.yaml`:
+
+<Details scope={["cloud"]} scopeOnly={true}>
+```yaml
+auth_server: "example.teleport.sh:443"
+onboarding:
+  join_method: "token"
+  token: "00000000000000000000000000000000"
```

Comment:
Instead of using a hard coded token and CA pin, we have preset variables for these.

See https://github.com/gravitational/teleport/pull/12868 for example.